src/server/api/routers/bot.ts

```diff
@@ -187,6 +192,39 @@ export const botRouter = createTRPCRouter({
       return bot
     }),
 
+  getBotAppearanceSettingByApiToken: integrationProcedure.query(
+    async ({ ctx }) => {
+      const source = await db.query.bots.findFirst({
+        columns: {
+          companyLogoAttachmentId: true,
+          botAvatarAttachmentId: true,
+          widgetName: true,
+          widgetSubheading: true,
+          widgetPlaceholder: true,
+          widgetWelcomeMsg: true,
+          accentColour: true,
+        },
+        where: eq(schema.bots.id, ctx.session.botId),
+      })
+
+      const data: BotAppearance = {
+        botId: ctx.session.botId,
+        companyLogoAttachmentId: source?.companyLogoAttachmentId ?? '',
+        botAvatarAttachmentId: source?.botAvatarAttachmentId ?? '',
+        widgetName: source?.widgetName ?? 'Dwarves Bot',
```

Comment:
Let's not define the default value here. Default value should only be defined when created. 
Or when show to user. Let's not make default value when get from db.

Reply:
You are right. It's not necessary to return the user.